metagraph/src/graph/alignment/aligner_chainer.hpp

```diff
@@ -0,0 +1,38 @@
+#ifndef __ALIGNER_CHAINER_HPP__
+#define __ALIGNER_CHAINER_HPP__
+
+#include "alignment.hpp"
+
+
+namespace mtg {
+namespace graph {
+namespace align {
+
+typedef std::vector<Alignment> Chain;
+typedef Alignment::score_t score_t;
+
+// Given sets of forward and reverse-complement seeds, construct and call chains of seeds
+// until terminate() returns true.
+std::pair<size_t, size_t>
+call_seed_chains_both_strands(std::string_view forward,
+                              std::string_view reverse,
+                              size_t node_overlap,
+                              const DBGAlignerConfig &config,
```

Comment:
Shouldn't `node_overlap` be a member of `DBGAlignerConfig`?

Reply:
I mean, one can do that, but I don't see how that's semantically more correct than what's happening here.